src/lifetracker/calendar/CalendarListImpl.java

```diff
@@ -29,19 +31,21 @@
      */
     @Override
     public List<CalendarEntry> getEventList() {
-        return eventList;
+        List<CalendarEntry> list = new ArrayList<CalendarEntry>(eventList.values());
+        return list;
     }
 
     /*
      * (non-Javadoc)
      *
-     * @see lifetracker.calendar.CalenderList#add(java.lang.String,
-     * java.time.LocalDateTime)
+     * @see lifetracker.calendar.CalenderList#add(java.lang.String)
      */
     @Override
     public void add(String name) {
-        CalendarEntryImpl ft = new CalendarEntryImpl(name, null, null);
-        taskList.add(ft);
+        int idToSet = taskList.lastKey() + 1;
```

Comment:
The id should be unique across both lists. You need to check both Maps for the last key.